site/en/codelabs/silabs-openthread-hardware/index.lab.md

```diff
@@ -0,0 +1,461 @@
+
+![Flash](./img/flash_device_selection.png)
+
+## Firmware set up summary
+
+Duration: 1:00
+
+At this point you should have created, compiled and flashed the appropriate firmware on the radio boards. Once `ot-rcp` is flashed to BRD4166A, disconnect it from the host system and connect this board to the Raspberry Pi.
+
+After completing this section, your Thread network hardware set up will look like this.
+
+> Not shown in the image is the WSTKs connected to a Windows/Linux/Mac Host system via USB and the Raspberry Pi 3B+ is connected to the internet using an Ethernet cable. 
+
+![EFR32MG set up](./img/EFR32MG_hardware_setup.jpg)
+
+
+
+## Set up the Serial console for `ot-cli-ftd` devices
+
+Duration: 1:00
+
+To launch the Console interface, in the Simplicity IDE perspective **right-click on your J-Link device** in the Devices View / Debug Adapters Window. Choose **Launch Console**. To get a prompt on the Console, choose **Serial 1** tab and **press Enter**. Check the state of the FTD node.
+
+> Note: For convenience, I also right clicked and renamed the debug adapters as `ot-cli-ftd 1` and `ot-cli-ftd 2`. 
+
+![FTD Studio Console View](./img/console_ftds.png)
+
+You will notice that we do not yet have a console for `ot-rcp`. In the next step we will configure the Raspberry Pi as an OT border router and set up the console for `ot-rcp`.
+
+## Set up the Raspberry Pi as a Border Router
+
+Duration: 30:00
+
+Silicon Labs recommends deploying the company’s Docker container with the OTBR. Running the OTBR in a container allows for creation of easily deployable artifacts and fast development prototyping and testing.
+
+Silicon Labs provides the following pre-built Docker containers (with tags), hosted on DockerHub:
+
+https://hub.docker.com/r/siliconlabsinc/openthread-border-router/tags
+
+https://hub.docker.com/r/siliconlabsinc/openthread-backbone-border-router/tags
+
+### Raspberry Pi set up
+
+1. On your SD card, make sure you flash the  [Raspbian Stretch Lite OS image](http://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2019-04-09/2019-04-08-raspbian-stretch-lite.zip) or [Raspbian Stretch with Desktop](https://downloads.raspberrypi.org/raspbian/images/raspbian-2019-04-09/2019-04-08-raspbian-stretch.zip).
+2. You can either SSH onto the Raspberry Pi or choose to work with the Raspbian Desktop directly. Open a terminal.
+3. Make sure to update your local repositories and package manager (**apt-get update** and **apt-get upgrade** prior to installing Docker).
+
+> Note: Make sure to reboot the Raspberry Pi after any updates.
```

Comment:
Add this before the note

<noparse>> aside positive</noparse>
<noparse>></noparse>